sentry/src/main/java/io/sentry/util/StringUtils.java

```diff
@@ -51,4 +54,22 @@ private StringUtils() {}
       return str;
     }
   }
+
+  /**
+   * Converts the given number of bytes to a human-readable string.
+   *
+   * @param bytes the number of bytes
+   * @return a string representing the human-readable byte count (e.g. 1kB, 20 MB, etc.)
+   */
+  public static @NotNull String byteCountToString(long bytes) {
+    if (-1000 < bytes && bytes < 1000) {
+      return bytes + " B";
+    }
+    CharacterIterator ci = new StringCharacterIterator("kMGTPE");
+    while (bytes <= -999_950 || bytes >= 999_950) {
+      bytes /= 1000;
+      ci.next();
+    }
+    return String.format(Locale.US, "%.1f %cB", bytes / 1000.0, ci.current());
```

Comment:
can it be `Locale.ROOT`? cant guarantee that `Locale.US` is available

Reply:
![image](https://user-images.githubusercontent.com/4999776/145538015-1dc145de-dcc6-488b-ad31-dc872d334bf6.png)
or are you aware of some edge case?

Reply:
it is an edge case, iirc Chinese ROMs